mappings/net/minecraft/entity/ai/goal/UniversalAngerGoal.mapping

```diff
@@ -0,0 +1,11 @@
+CLASS net/minecraft/class_5398 net/minecraft/entity/ai/goal/UniversalAngerGoal
+	FIELD field_25604 entityType Lnet/minecraft/class_1308;
+	FIELD field_25605 angerOthers Z
```

Comment:
??? why "others" if the anger is "universal"

Reply:
Universal anger, per the gamerule, means that an angry mob will be angry at every player rather than just the one that attacked them. It does not mean that every mob will be angry. This method is used for mobs that attack in groups (bees, wolves, zombie piglins) to notify others in range to get mad (at no one in particular, just get mad).

Reply:
maybe `triggerOthers` then.

Reply:
```suggestion
	FIELD field_25605 triggerOthers Z
```